homeassistant/components/vicare/sensor.py

```diff
@@ -306,6 +311,31 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
     ),
 )
 
+SOLAR_SENSORS: tuple[ViCareSensorEntityDescription, ...] = (
```

Comment:
could you please add those sensors to GLOBAL_SENSORS?
we don't separate the sensors by heating device type but by how they are structured on PyVicare.